### ISSUE TITLE
Markdown renderer

### ISSUE THREAD

**Issue by kno10** [plastex/plastex#273]
IMHO, a Markdown renderer would be a great addition, allowing Plastex to be easily used for latex -> markdown conversion, without going through HTML etc. first.

Yes, I know that there are many markdown dialects, unfortunately.

**Comment by PatrickMassot** [comment]
Having something that works for a reasonable subset of TeX should be doable. I see from your other issues that you may be trying to adapt the text renderer, but I think it would be more efficient to subclass the page template renderer rather than using the barebone renderer API.

**Comment by kno10** [comment]
I have a basic markdown renderer using PageRenderer with jinjia2 templates in this branch:
https://github.com/kno10/plastex/tree/markdown

**Comment by PatrickMassot** [comment]
Feel free to open a pull-request with this renderer, indicating clearly whether it depends on other pull-requests (since I see that branch also has unrelated commits).

**Comment by kno10** [comment]
Yes, this currently is not a clean branch, because I have to fix various other issues along the way.
Some of these (such as the whitespace problem #313) possibly block this output format, too.
I mostly referenced it for others as a possible starting point if they also want to produce Markdown for now.
